src/components/InputField/InputField.stories.mdx

```diff
@@ -96,7 +96,7 @@ export default InputFieldExample;
 | name           | specifies the name                                                                       | `string`                                                                    |
 | isDisabled     | disable state                                                                            | `bool`                                                                      |
 | placeholder    | defines placeholder for the field                                                        | `string`                                                                    |
-| scrollIntoView | -                                                                                        | `bool`                                                                      |
+| scrollIntoView | scrolls an element into the viewport                                                     | `bool` &vert; `object`                                                      |
```

Comment:
Why do we have `&vert;` can't we escape it and use ?

Reply:
Yes, we can